src/main/java/org/folio/circulation/domain/policy/library/ClosedLibraryStrategyService.java

```diff
@@ -48,6 +49,16 @@ public ClosedLibraryStrategyService(
       .thenApply(mapResult(relatedRecords::withLoan));
   }
 
+  public CompletableFuture<Result<RenewalContext>> applyClosedLibraryDueDateManagement(
+    RenewalContext renewalContext) {
+
+    final Loan loan = renewalContext.getLoan();
+    return applyClosedLibraryDueDateManagement(loan, loan.getLoanPolicy(),
+      renewalContext.getTimeZone())
+      .thenApply(mapResult(loan::changeDueDate))
+      .thenApply(mapResult(renewalContext::withLoan));
+  }
+
```

Comment:
Please could the line formatting of the overloaded method below it also be corrected